pyproject.toml

```diff
@@ -21,12 +21,12 @@ eq-translations = {git = "https://github.com/ONSDigital/eq-translations.git"}
 [tool.poetry.dev-dependencies]
 pytest = "^6.2.5"
 flake8 = "^4.0.1"
-pylint = "^2.12.2"
+pylint = "^2.15.3"
 pylint-quotes = "^0.2.3"
 pytest-cov = "^3.0.0"
 pytest-xdist = "^2.5.0"
 pytest-sugar = "^0.9.4"
-black = "21.10b0"
+black = "^21.10b0"
```

Comment:
You might have to pin it to `"^22.10.0"` here since the previous pin will only update up to latest `21` version.

Reply:
updated, thanks !